src/index.js

```diff
@@ -35,6 +35,22 @@ client.getCommand = function(name) {
 
 require("./commands")(client);
 
+/**
+ * Checks to see if there is a different command similar to the provided one.
+ * This prints the suggestion and returns it if there is one.
+ * @param {string} cmd The command as provided by the user.
```

Comment:
I didn't know about putting the curly braces around the type here, what a nice way to provide some light type info in js :)